cmd/minikube/cmd/root.go

```diff
@@ -98,6 +99,17 @@ func Execute() {
 		} else {
 			os.Args = append([]string{RootCmd.Use, callingCmd, "--"}, os.Args[1:]...)
 		}
+	} else if filepath.Ext(callingCmd) == ".exe" && driver.IsMicrosoftWSL() {
+		var found = false
+		for _, a := range os.Args {
```

Comment:
I am curious why wouldn't we just use viper.GetBool(force) 
for example 	if !viper.GetBool(force) { 
}
https://github.com/medyagh/minikube/blob/446e0f0946f820fe612dfdda5d7b1a34d2b7b36f/cmd/minikube/cmd/start.go#L1258

Reply:
That was my first attempt, and it fails. It seems in here, the viper has not been initialized, so using `viper.GetBool(force)` cannot retrieve the flag value.